.github/actions/run_examples_using_pypi_uploads/action.yml

```diff
@@ -0,0 +1,38 @@
+name: Run examples using pypi uploads
+description: Install module(s) from PyPI and run examples. Intended for release validation.
+inputs:
+  module_name:
+    description: module name
+    required: true
+    default: ""
+  module_version:
+    description: module version
+    required: true
+    default: ""
+
+runs:
+  using: composite
+  steps:
+    # The local wheel installation and single environment would have made using tox-system_tests.ini difficult.
+    # To keep things simple, we just use pip and pytest directly for a single Python version.
+    - run: py -3.12 -m venv --clear .venv
```

Comment:
can you think of any way to make it less likely that this version becomes stale as new Python releases come out?
any way to single source?

(if it's too complicated, ok to leave alone)

Reply:
Maybe it would help if Python version was an input as well?

Reply:
Well, when I add a new version, I generally do a search for the last version, (3.12, 312), so I expect it won't be a problem. Version support changes are currently completely manual. It's one more thing we could stand to have a tool for.

Reply:
I created #2060 to help address your concerns.